packages/website/posts/2022-01-25-what-am-buying.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: 'When you buy an NFT, what are you actually getting?'
+description: Behind the glamour, NFTs are more like hyperlinks than like files.
+author: J Chris Anderson
+thumbnail: https://user-images.githubusercontent.com/22168118/149502173-d50d7e98-29a9-4c53-9dbc-ed19ab0ae755.png
+date: Jan 25, 2022
+tags:
+  - culture
+---
+
+When you buy an NFT, you are purchasing your name at the top of the list of holders. The value comes from the previous holders (and the link to the creator), more than the image, video or other asset. You are also buying the right to decide who is the next token holder. You could sell it or give it away.
+
+None of this has much to do with the copyright of the linked artwork. Does a spam NFT linking to a stolen image violate copyright? What about users minting “their” version of a famous meme, and trading the NFT with friends? By setting aside concerns about the attached content (image, video, html), we can see that the power is in the link itself.
+
+NFTs are not about the scarcity of the linked image, they are a new type of link. A link that memorializes how it is shared through a community, and becomes valuable because of its history. A link that gains its uniqueness from human interactions.
+
+There is enough power in this new kind of link, just because it tracks how it changed hands, to enable a market. **A market of links!**
+
+Where does the value come from?
+
+(This post is based on a twitter thread you can find here.)
+
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">When you buy an NFT, what are you actually getting?</p>&mdash; index.html (@jchris) <a href="https://twitter.com/jchris/status/1485664899862708225?ref_src=twsrc%5Etfw">January 24, 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The authentic link to the creator and the community of buyers, is encoded in that cryptographic list of holders. Without it, the NFT loses value. In the extreme case, machines link to an unsuspecting artist’s image, and make money from uneducated buyers. As more people understand that what they’re buying is a link with the artist and community, not the image itself, the public will become more educated about what’s worth buying.
+
+Tools owe it to users to make validating the provenance of NFTs a core part of the experience. If my pfp is legendary because it used to be owned by a celebrity, I want the world to know. Those same tools will help devalue spam, and put guardrails to prevent accidentally purchasing stolen NFTs. Over time, people will learn what makes a good NFT, and scammers will find another playground.
+
```

Comment:
Maybe worth mentioning how this isn't much different than physical art - originals vs. prints vs. counterfeits?